pkgs/development/interpreters/gnudatalanguage/default.nix

```diff
@@ -0,0 +1,167 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+# Native build inputs for the compiler
+, cmake
+, wrapGAppsHook
+# Build dependencies
+, readline
+, ncurses # alt: curses
+, zlib
+, gsl
+, openmp
+# GraphicsMagick are preferred over ImageMagick
+# https://github.com/gnudatalanguage/gdl/blob/master/CMakeLists.txt#L600-L601
+, graphicsmagick # alt: imagemagick
+, netcdf
+, hdf4
+, hdf5
+, fftw, fftwFloat, fftwLongDouble
+, proj
+, shapelib
+, expat
+, udunits
+, eigen
+, pslib # a C library for PostScript
+, eccodes
+, glpk
+, libpng
+, plplot
+, libtiff
+, libgeotiff
+, libjpeg # for hdf4 and hdf5
+, enableLibtirpc ? stdenv.isLinux, libtirpc
+, pythonInterpreter ? python3, python3
+, enableMPI ? (stdenv.isLinux || stdenv.isDarwin)
+# Choose mpich because of it builds on aarch and darwin
+, libraryMPI ? mpich # alt: openmpi
```

Comment:
It should be a string, shouldn't?

Reply:
`python3` means the python3 package (and the source of NumPy). I'm not sure if it's redundant, but it offers an option to specify the python interpreter to use with GDL.

Reply:
Updated.